src/main/java/hu/oe/nik/szfmv/automatedcar/systemcomponents/Sensor.java

```diff
@@ -25,18 +26,17 @@
     private static final Logger LOGGER = LogManager.getLogger(Main.class);
     private static final int RADAR_RANGE_IN_METER = 3;
     private static final double RADAR_SENSOR_ANGLE = Math.PI / 180 * 50;// 50°
-    private static final Color RADAR_SENSOR_BG_COLOUR = new Color(255, 0, 0, 125);
+    //  private static final Color RADAR_SENSOR_BG_COLOUR = new Color(255, 0, 0, 125);
```

Comment:
A `color` beállítás az `UltasoundsVisualizationPacket`-ben van megírva, itt majd lehet meg kell változtatni a színt (példában zöld)